travis-int-test.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# exit when any command fails
+set -e
+
+echo "Running integration tests"
+./gradlew -s --scan intTest codeCoverageReport
+
+echo "Uploading integration test coverage to codecov"
+bash <(curl -s https://codecov.io/bash)
```

Comment:
This script searches for the coverage report in the project build directory, detects what type it is and then uploads it to the codecov server. This used to be in the after_success part of yaml file but had to be moved since adding the multiple jobs.

More details on the script [here](https://docs.codecov.io/docs/about-the-codecov-bash-uploader)